pkgs/tools/security/modsecurity/default.nix

```
@@ -37,7 +39,7 @@ stdenv.mkDerivation rec {
   outputs = ["out" "nginx"];
   # by default modsecurity's install script copies compiled output to httpd's modules folder
   # this patch removes those lines
-  patches = [ ./Makefile.in.patch ];
+  patches = [ ./Makefile.am.patch ];
 
```

```suggestion
  doCheck = true;
  checkInputs = [ perl ];
```

WFM linux and macos 10.15 and gives a little confidence in a package, let's be honest, neither of us are going to test fully.

---

Thanks for that!